docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#change-or-delete-response-headers' %}
+
+### Change or Delete Response Headers
+
+The `RequestHook._onConfigureResponse()` method sets and removes headers, effectively mocking part of the response.
+
+To change response headers, use `event.setHeaders` and `event.removeHeader` methods in the `__onConfigureResponse` call.
+
+> Important! The `RequestHook._onConfigureResponse()` method is private. Be cautious when you add this method to your request hooks.
```

Comment:
More technically correct:

`Be cautious when you override this method to your request hooks.`